### DISCUSSION TITLE
引入 iconfont 图标失效问题

### DISCUSSION THREAD

**Discussion by BraumAce** [vuepress-theme-hope/vuepress-theme-hope#4266] [Q&A] [answered]
如下，通过 iconAssets 设置图标资源后，在导航栏和文章 Frontmatter 配置中引入图标均不显示
无论图标资源 url 前是否加了 https: 均不显示
![iconAssets](https://github.com/user-attachments/assets/df591a9a-a8df-4d60-aed1-205695bac5cb)

导航栏引用：
![image](https://github.com/user-attachments/assets/a187b39c-83f2-4b7c-bea3-d701d6f60e31)

文章标题引用：
![image](https://github.com/user-attachments/assets/3bccaf24-6393-4269-8bd1-add0faae1365)

图标不显示：
![image](https://github.com/user-attachments/assets/f1a3d2b4-a131-467c-b9b4-1eba1081c768)
![image](https://github.com/user-attachments/assets/ebf6614a-214f-4c22-b980-001a7e440ec1)

但将生成的 fontclass 代码拷贝到 index.scss 中，可以正常显示，如下：
![fontclass](https://github.com/user-attachments/assets/2c6000dd-e15c-4df2-9a06-384e5f754784)

图标正常显示：
![image](https://github.com/user-attachments/assets/0a22875c-b946-41f2-9cd2-aca6bd841bd1)

![image](https://github.com/user-attachments/assets/b1aea048-814b-4cbf-bb87-5482a2828ba2)

**Comment by Mister-Hope** [comment, marked as answer]
A reproduction is needed, when you are ready open an issue instead.

Iconfont is working fine with https://mister-hope.com/ and https://waline.js.org
